python/cudf/bindings/replace.pyx

```diff
@@ -30,6 +30,11 @@ cpdef replace(col, old_values, new_values):
 
     gdf_find_and_replace_all(c_col, c_old_values, c_new_values)
 
+    cpdef nnz
```

Comment:
Is this a function definition? If so where's the argument list and the colon? Is this syntactically correct?

Reply:
This is variable declaration. In cython, C-type variable can not be declared with in a block.